tools/ci_build/github/azure-pipelines/orttraining-pai-ci-pipeline.yml

```diff
@@ -1,33 +1,38 @@
 trigger: none
 
 name: 'orttraining_ci_$(Date:yyyyMMdd)_$(Rev:r)'
-
-pool: 'Linux-CPU-CorpNet'
-
-variables:
-- group: pai-variables
-- name: pai-user
-  value: onnxrt
+pool: 'AMD-GPU'
 
 steps:
 - checkout: self
   clean: true
 
 - script: |-
-    python3 -m pip install -r tools/ci_build/github/pai/requirements.txt
-  displayName: 'Install requirements'
+    echo "##vso[task.prependpath]/home/ciagent/conda/bin/"
+    echo "##vso[task.prependpath]/home/ciagent/pkg/openmpi-4.0.5/bin/"
+    echo '##vso[task.setvariable variable=LD_LIBRARY_PATH]/home/ciagent/pkg/openmpi-4.0.5/lib/'
+    eval "$('/home/ciagent/conda/bin/conda' 'shell.bash' 'hook' 2> /dev/null)"
+    echo "Selecting GPU based on HIP_VISIBLE_DEVICES=$HIP_VISIBLE_DEVICES"
+  displayName: 'Initialize environment'
 
 - script: |-
-    python3 tools/ci_build/github/pai/run_job.py \
-      tools/ci_build/github/pai/orttraining-ci.yml $(Build.BuildNumber) \
-      --user-env PAI_USER --token-env PAI_TOKEN \
-      --yaml-sub-env docker_user DOCKER_USER \
-      --yaml-sub-env docker_password DOCKER_TOKEN \
-      --yaml-sub-env commit BUILD_SOURCEVERSION
-  displayName: 'Run PAI job'
-  env:
-    # map secret variables to environment variables
-    PAI_USER: $(pai-user)
-    PAI_TOKEN: $(pai-token)
-    DOCKER_USER: $(pai-docker-user)
-    DOCKER_TOKEN: $(pai-docker-token)
+    python tools/ci_build/build.py \
+    --config RelWithDebInfo \
+    --enable_training \
+    --mpi_home /home/ciagent/pkg/openmpi-4.0.5 \
+    --use_rocm \
+      --rocm_home /opt/rocm-3.9.0 \
+      --nccl_home /opt/rocm-3.9.0 \
+    --update \
+    --build_dir ./build \
+    --build \
+    --parallel 8 \
+    --build_wheel \
+    --skip_tests
+  displayName: 'Build onnxruntime'
+
+- script: |-
+    cd ./build/RelWithDebInfo &&\
```

Comment:
fyi, there is a workingDirectory parameter
https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=azure-devops&tabs=schema%2Cparameter-schema#script

Reply:
OK, good to know. I think I'll incorporate on next revision (more to come as tests are brought back..)